datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -1,110 +1,411 @@
+    font-size: 20px;
+    line-height: 28px;
+    color: #262626;
+    margin: 13px 0 7px 0;
+`;
+const UserRole = styled.div`
+    font-size: 14px;
+    line-height: 22px;
+    color: #595959;
+    margin-bottom: 7px;
+`;
+const UserTeam = styled.div`
+    font-size: 12px;
+    line-height: 20px;
+    color: #8c8c8c;
+`;
+const UserSocialDetails = styled.div`
+    font-size: 12px;
+    line-height: 20px;
+    color: #262626;
+    text-align: left;
+    margin: 6px 0;
+`;
+const EditProfileButton = styled.div`
+    // margin-bottom: 24px;
+    bottom: 24px;
+    position: absolute;
+    right: 27px;
+    width: 80%;
+    left: 50%;
+    -webkit-transform: translateX(-50%);
+    -moz-transform: translateX(-50%);
+    transform: translateX(-50%);
+
+    button {
+        width: 100%;
+        font-size: 12px;
+        line-height: 20px;
+        color: #262626;
+    }
+`;
+const AboutSection = styled.div`
+    text-align: left;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 22px;
+    color: #262626;
+`;
+const AboutSectionText = styled.div`
+    font-size: 12px;
+    font-weight: 100;
+    line-height: 15px;
+    padding: 5px 0;
+
+    &&& .ant-typography {
+        margin-bottom: 0;
+    }
+    &&& .ant-typography-edit-content {
+        padding-left: 15px;
+        padding-top: 5px;
+    }
+`;
+const GroupsSection = styled.div`
+    text-align: left;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 22px;
+    color: #262626;
+`;
+const TagsSection = styled.div`
+    height: calc(75vh - 460px);
+    padding: 5px;
+    // overflow: auto;
+`;
+const NoDataFound = styled.span`
+    font-size: 12px;
+    color: #262626;
+    font-weight: 100;
+`;
+const Tags = styled.div`
+    margin-top: 5px;
+`;
+const GroupsSeeMoreText = styled.span`
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 20px;
+    color: #1890ff;
+    cursor: pointer;
+`;
+const Content = styled.div`
+    color: #262626;
+    height: calc(100vh - 60px);
+
+    &&& .ant-tabs > .ant-tabs-nav .ant-tabs-nav-wrap {
+        padding-left: 15px;
+    }
+`;
+
 /**
  * Responsible for reading & writing users.
  */
 export default function UserProfile() {
     const { urn: encodedUrn } = useUserParams();
     const urn = decodeUrn(encodedUrn);
-    const { loading, error, data } = useGetUserQuery({ variables: { urn, groupsCount: GROUP_PAGE_SIZE } });
+    const [updateCorpUserPropertiesMutation] = useUpdateCorpUserPropertiesMutation();
+    const { loading, error, data, refetch } = useGetUserQuery({ variables: { urn, groupsCount: GROUP_PAGE_SIZE } });
     const entityRegistry = useEntityRegistry();
-    const username = data?.corpUser?.username;
 
+    const username = data?.corpUser?.username;
     const ownershipResult = useGetAllEntitySearchResults({
         query: `owners:${username}`,
     });
 
+    const [groupSectionExpanded, setGroupSectionExpanded] = useState(false);
+    const [editProfileModal, setEditProfileModal] = useState(false);
```

Comment:
Let's rename to "showEditProfileModal"

Reply:
Sure, required variable text name change from `setEditProfileModal` to `showEditProfileModal`

Reply:
(I meant both)